pipelines/batch/src/main/java/com/google/fhir/analytics/FhirSearchUtil.java

```diff
@@ -317,6 +332,88 @@ Bundle searchByPatientAndLastDate(
     return query.execute();
   }
 
+  /**
+   * Fetch the details of bulk export job for the given bulkExportStatusUrl
+   *
+   * @param bulkExportStatusUrl the url of the bulk export job
+   * @return BulkExportHttpResponse - the status and details of the bulk export job
+   * @throws IOException
+   */
+  public BulkExportHttpResponse fetchBulkExportHttpResponse(String bulkExportStatusUrl)
+      throws IOException {
+    IHttpResponse httpResponse = fetchUtil.fetchResponseForUrl(bulkExportStatusUrl);
+    BulkExportHttpResponse bulkExportHttpResponse = new BulkExportHttpResponse();
+    bulkExportHttpResponse.setHttpStatus(httpResponse.getStatus());
+    if (httpResponse.getHeaders("expires") != null
+        && !httpResponse.getHeaders("expires").isEmpty()) {
+      String expiresString = httpResponse.getHeaders("expires").get(0);
```

Comment:
Is there anyway to tell the server that we need the files for at least a certain period? If not, what is HAPI's default and how can it be changed? I suppose it is a server configuration that we should document too.

Reply:
I had a look at this, here are my findings
- The client cannot control the expiry time.
- The expiry time is server dependent and by default it is 2 hours for [HAPI server](https://github.com/hapifhir/hapi-fhir/blob/e07684474f75b3bad462e8d8e4045c1691c1fc5e/hapi-fhir-storage/src/main/java/ca/uhn/fhir/jpa/api/config/JpaStorageSettings.java#L2234). I could not find any config parameter which can be set at the server side though.

Reply:
Great, thanks for the investigation. The 2 hour limit is a concern, especially if it is not configurable. It would be great if you can file an issue against HAPI for adding the configuration parameter. And also please make sure that we document your findings somewhere more visible.

Reply:
Sure, will be filing the ticket. I will check where I can document this.

Reply:
Created this [issue](https://github.com/hapifhir/hapi-fhir-jpaserver-starter/issues/710)